src/duke/Duke.java

```diff
@@ -1,4 +1,4 @@
-import main.java.*;
+package duke;
```

Comment:
You might want to consider more packages, such as a util package for the "tool" classes (e.g. Ui, Storage, Parser) and another package for Tasks (Deadline, ToDo, Task etc.)